src/main.ts

```diff
@@ -79,6 +80,7 @@ const init = async (): Promise<void> => {
     app.use(helmet());
     app.use(hpp());
     app.get('/health', (_: Request, res: Response) => res.sendStatus(200));
+    app.get('/config', (_: Request, res: Response) => res.json({ config: clientConfig }));
```

Comment:
I think the endpoint had an extension in xPug - not that it matters as long as we have the correct mimetype

Reply:
xPub was sending a JS and adding window.config. Not sure that's wise.